source/extensions/transport_sockets/tls/cert_validator/default_validator.cc

```diff
@@ -551,15 +551,26 @@ Envoy::Ssl::CertificateDetailsPtr DefaultCertValidator::getCaCertInformation() c
   return Utility::certificateDetails(ca_cert_.get(), getCaFileName(), time_source_);
 }
 
+void DefaultCertValidator::refreshCertStatsWithExpirationTime() {
+  auto expiration_unix_time = Utility::getExpirationUnixTime(ca_cert_.get());
+  if (expiration_unix_time.has_value()) {
```

Comment:
if the expiration time has no value, does that mean it's a non-expiring cert? Should we emit something for that, like (say) '0'?